### ISSUE TITLE
error running the selected code generator index out of bounds

### ISSUE THREAD

**Issue by vibhavpan** [MicrosoftDocs/feedback#3866]
I was following the razor course ,but when I am trying to add a new scaffolding item,it is giving the above error.Please help

**Comment by sugarbush1111** [comment]
Thank you all so must its a shame that anyone would have to experience that
Happy Holidays

**Comment by guardrex** [comment]
@sugarbush1111 ... Open an issue from the bottom of the "razor course" article (tutorial?). Use the **This page** feedback button and form at the bottom of the **English-US** topic. Use of the **This page** feedback form adds metadata to your GitHub issue that cross-links the topic and automatically pings the author.

<img width="404" alt="Capture" src="https://user-images.githubusercontent.com/1622880/81926733-0b02d200-95a8-11ea-8683-8dadc5483fb1.PNG">

**Comment by gewarren** [comment]
Could be from a training module like https://learn.microsoft.com/en-us/training/modules/create-razor-pages-aspnet-core/?

**Comment by guardrex** [comment]
Indeed, yes. I don't have any direct contact with the doc authors working on those. Scott and Cam were working on some of them, but I don't know if that's one of their efforts.

**Comment by sugarbush1111** [comment]
Thank you
